lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -19,10 +19,14 @@
  *
  * @link https://core.trac.wordpress.org/ticket/53494.
  *
+ * @deprecated 6.1 Block supports styles are now stored for enqueuing via the style engine API. See: packages/style-engine/README.md.
+ *
  * @param string $style String containing the CSS styles to be added.
  * @param int    $priority To set the priority for the add_action.
  */
 function gutenberg_enqueue_block_support_styles( $style, $priority = 10 ) {
+	_deprecated_function( __FUNCTION__, '6.1' );
```

Comment:
I notice that with this PR applied, this is being logged once per page request:

<img width="1339" alt="image" src="https://user-images.githubusercontent.com/14988353/183358421-c64730fe-6bf8-49cf-b5f4-60639b5cbff3.png">

It looks like it's being called from the Gallery block. I imagine we should update that call, too, otherwise we'll potentially risk flooding logs of sites running the GB plugin?

Alternately, we could leave the `_deprected_function` call to a follow-up if you'd prefer.

Reply:
Will do, thanks!

Reply:
> deferring introducing the _deprecated_function call to a follow-u

Actually, I think this you had the best idea for now. We can deprecate when we deal with the final Gutenberg usage in the gallery.

Just in case the gallery gets hairy 😄 

I've reverted this change.